src/moduleManager.js

```diff
@@ -62,6 +65,7 @@ export const importModules = async (files, isLocal) => {
   }
 
   goosemodScope.logger.debug('import', 'Imported all files');
+  goosemodScope.showToast("Module(s) successfully imported", { type: "success" });
```

Comment:
This is mostly unneeded as it'll have a toast for each module with the above changes anyway.